src/senaite/core/content/base.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+from AccessControl import ClassSecurityInfo
+from bika.lims import api
+from plone.dexterity.content import Container as BaseContainer
+from plone.dexterity.content import Item as BaseItem
+from senaite.core.interfaces import IContainer
+from senaite.core.interfaces import IItem
+from zope.interface import implementer
+
+
+@implementer(IContainer)
+class Container(BaseContainer):
+    """Base class for SENAITE folderish contents
+    """
+    security = ClassSecurityInfo()
+
+    @security.private
+    def accessor(self, fieldname):
+        """Return the field accessor for the fieldname
+        """
+        schema = api.get_schema(self)
+        if fieldname not in schema:
+            return None
+        return schema[fieldname].get
+
+    @security.private
+    def mutator(self, fieldname):
+        """Return the field mutator for the fieldname
+        """
+        schema = api.get_schema(self)
+        if fieldname not in schema:
+            return None
+        return schema[fieldname].set
+
+
+@implementer(IItem)
+class Item(BaseItem):
```

Comment:
Don't remember why atm, but I think we decided to always make our DX types to inherit from `Container` right?

Reply:
Yes, that's true. It's there just for completeness.